exercises/resistor-color-trio/canonical-data.json

```
@@ -56,6 +56,17 @@
         "value": 470,
         "unit": "kiloohms"
       }
+    },
+     {
```

```suggestion
    {
```
Remove single extra leading space.